tuxemon/cli/commands/print.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from tuxemon.cli.clicommand import CLICommand
+from tuxemon.cli.context import InvokeContext
+
+class PrintCommand(CLICommand):
+    """
+    Print variables set by map event actions.
+
+    """
+
+    name = "print"
+    description = "Print values of variables set using map actions and events."
+    example = "print steps"
+
+    def invoke(self, ctx: InvokeContext, line: str) -> None:
+        """
+        Print values of variables set using map actions and events.
+
+        Parameters:
+            ctx: Contains references to parts of the game and CLI interface.
+            line: Complete text as entered into the prompt.
+
+        """
+        variable = line.strip()
+        if variable > "":
```

Comment:
lets change this to just `if variable` because the `>` when used with a string is idk, really not used much and could raise some eyebrows.

Reply:
Thank you, I've changed it from if variable > "": to if variable: - I'll try to remember to do that in the future, too. Thanks!